features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,11 @@
+Given("I visit the landing page") do
+  visit root_path
+end
+  
+Then("I should see {string}") do |content|
+  expect(page).to have_content content 
+end
+  
+Then("I click {string} link") do |link|
```

Comment:
add this step to the assertions steps instead

Reply:
on it

Reply:
OK. Added ;)